src/core/Akka.MultiNodeTestRunner.TrxReporter/Akka.MultiNodeTestRunner.TrxReporter.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\common.props" />
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkLibVersion);$(NetStandardLibVersion)</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
```

Comment:
Should set this to a specific value so we don't accidentally jump onto C# 8, which includes some breaking changes (`static interface`) - although I don't think targeting C# 8 by itself will create a breaking change with our DLLs. It's only if we use `static interface` that this will come up.